egs/multi_conv_zh_es_ta/ST/zipformer_multijoiner_st/decode.py

```
@@ -1071,25 +1074,25 @@ def main():
     test_iwslt22_dl = multiling.test_dataloaders(test_iwslt22)
     test_fisher_dl = multiling.test_dataloaders(test_fisher)
 
-    test_sets = [ "test-fisher", "iwslt-ta", "test-hkust"]
+    test_sets = ["test-fisher", "iwslt-ta", "test-hkust"]
 
     test_dl = [test_fisher_dl, test_iwslt22_dl, test_hkust_dl]
 
     for test_set, test_dl in zip(test_sets, test_dl):
```

Ruff (0.15.18)

[warning] Loop control variable `test_dl` overrides iterable it iterates

(B020)